src/routes/verify/[verifyToken]/+page.svelte

```diff
@@ -21,10 +21,10 @@
     };
 
     const serverErrorToast = {
-        message: "Fehler auf der Serverseite",
+        message: "Verfizierung nicht möglich. Versuche es später erneut",
```

Comment:
```suggestion
        message: "Verfizierung nicht möglich. Versuche es später erneut.",
```
Hatten wir glaube ich sonst auch so... wenn irgendwo n Punkt drin ist soll auch einer ans Ende